pandas/tests/series/test_operators.py

```diff
@@ -1997,25 +2003,39 @@ def test_series_radd_str(self):
         assert_series_equal(ser + 'a', pd.Series(['xa', np.nan, 'xa']))
 
     @pytest.mark.parametrize('dtype', [None, object])
-    def test_series_radd_more(self, dtype):
-        res = 1 + pd.Series([1, 2, 3], dtype=dtype)
-        exp = pd.Series([2, 3, 4], dtype=dtype)
-        assert_series_equal(res, exp)
-        res = pd.Series([1, 2, 3], dtype=dtype) + 1
-        assert_series_equal(res, exp)
-
-        res = np.nan + pd.Series([1, 2, 3], dtype=dtype)
-        exp = pd.Series([np.nan, np.nan, np.nan], dtype=dtype)
-        assert_series_equal(res, exp)
-        res = pd.Series([1, 2, 3], dtype=dtype) + np.nan
-        assert_series_equal(res, exp)
-
-        s = pd.Series([pd.Timedelta('1 days'), pd.Timedelta('2 days'),
-                       pd.Timedelta('3 days')], dtype=dtype)
-        exp = pd.Series([pd.Timedelta('4 days'), pd.Timedelta('5 days'),
-                         pd.Timedelta('6 days')])
-        assert_series_equal(pd.Timedelta('3 days') + s, exp)
-        assert_series_equal(s + pd.Timedelta('3 days'), exp)
+    def test_series_with_dtype_radd_timedelta(self, dtype):
+        ser = pd.Series([pd.Timedelta('1 days'), pd.Timedelta('2 days'),
+                         pd.Timedelta('3 days')], dtype=dtype)
+        expected = pd.Series([pd.Timedelta('4 days'), pd.Timedelta('5 days'),
+                              pd.Timedelta('6 days')])
+
+        result = pd.Timedelta('3 days') + ser
+        assert_series_equal(result, expected)
+
+        result = ser + pd.Timedelta('3 days')
+        assert_series_equal(result, expected)
+
+    @pytest.mark.parametrize('dtype', [None, object])
```

Comment:
at the very least should split test_operators to timestamp, timedelta, numeric sub-modules (next PR)